installer/manifests/keptn/charts/control-plane/templates/api-gateway-nginx.yaml

```diff
@@ -55,6 +55,8 @@ data:
           listen       8080;
           server_name  _;
 
+          client_max_body_size {{ .Values.apiGatewayNginx.clientMaxBodySize | default "1m" }};
```

Comment:
We should make sure this value exists (e.g., what if `.Values.apiGatewayNginx` is `nil`)

Reply:
I refactored it such that the `client_max_body_size` directive will only be printed if the helm value is defined.